std/js/html/FormData.hx

```diff
@@ -43,6 +43,10 @@ extern class FormData
 	**/
 	@:overload( function( name : String, value : Blob, ?filename : String ) : Void {} )
 	function append( name : String, value : String ) : Void;
+	
+	/**
+		Deletes a key/value pair from a `FormData` object.
+	**/
 	@:native("delete")
 	function delete_( name : String ) : Void;
```

Comment:
Hm, not related to this particular change, but `delete` is not a keyword in Haxe, so this shouldn't be needed.

Reply:
Good catch @nadako! Not sure where the original keyword list is from but I'll synchronize it with `ast.ml` and update this PR